features/support/env.rb

```diff
@@ -0,0 +1,15 @@
+require 'aruba/cucumber'
+require 'komenda'
+
+Aruba.configure do |config|
+  config.exit_timeout = 300
+  config.activate_announcer_on_command_failure = [:stdout, :stderr]
+end
+
+After do |_scenario|
+  Komenda.run('bundle exec vagrant landrush stop', fail_on_fail: true)
```

Comment:
It's just a simple way to run a shell command. I couldn't easily find something exposed by Aruba (which might be nicer?).